### ISSUE TITLE
Link hover state does not meet contrast requirements on font sizes below 18px

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#987]
> When you hover over links, the combination of colours is #2b8cc4 on white. That is a contrast ratio of 3.72 which is less then the 4.5 that is required by WCAG AA.

> See [Understanding SC 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) to get confirmation that the minimum contrast ratio also applies to hover states.

Originally raised by @selfthinker on GOV.UK Elements: https://github.com/alphagov/govuk_elements/issues/546

**Comment by 36degrees** [comment]
This was also flagged in an audit of the design system (5th June 2018):

> "The pages contain underlined low contrast links that provide little notification when mouse pointer passes over. As a vision impaired user, additional visual cue is needed to confirm mouse pointer and link position.
> 
> "The visual cue, such as the removal of an underlining or background colour change on buttons, would offer a change in appearance confirming functionality. This helps to speed up navigation and provides greater user interaction with the page.
>
> "Currently, light blue is used to display links which is difficult to see despite meeting minimum AA guidelines. This maybe insufficient contrast suitable for all readers." (DAC-USER-LV-GT-T1-02)

**Comment by kellylee-gds** [comment]
Closed issue as its in the done column on the Kanban board
